src/symbolic_dimensions.jl

```diff
@@ -1,6 +1,5 @@
 import .Units: UNIT_SYMBOLS, UNIT_MAPPING, UNIT_VALUES
 import .Constants: CONSTANT_SYMBOLS, CONSTANT_MAPPING, CONSTANT_VALUES
-import SparseArrays as SA
 
 const SYMBOL_CONFLICTS = intersect(UNIT_SYMBOLS, CONSTANT_SYMBOLS)
 
```

Comment:
```suggestion
const INDEX_TYPE = UInt16
```

So we only need to set it in one location.

UInt16 over UInt8 to give more room for when more symbols are added in the future. 255 seems too little and could cause overflow issues if someone is not aware and adds their own set of symbols.

Reply:
Actually wait I'm having second thoughts. Maybe `UInt8` is good! We're only at 113 symbols right now.

That being said it would still be nice to have this set in a global constant.

Reply:
Yes, I was considering adding a global constant as well, I think that'll make it easier to change the type if needed.

```suggestion

const INDEX_TYPE = UInt8
```